app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -103,7 +103,7 @@ ListingController = (
   $scope.imageURL = (listing) ->
     # TODO: remove "or" case when we know we have real images
     # just a fallback for now
-    listing.Building_URL || 'https://placehold.it/474x316'
+    listing.imageURL || 'https://placehold.it/474x316'
 
```

Comment:
I like keeping this as a fallback, just in case we ever had a problem with images on production, either because of a missing/malformed image url, or because we couldn't process (resize and optimize) the image for some reason. That way the layout would still flow okay until the problem was resolved and the cache refreshed.